### PR TITLE
WB-599 feat(internal): Update docs to use import instead of require

### PR DESCRIPTION
**Pull request by jandrade**
## Summary
- Updated React Styleguidist examples to use `import` instead of `require`.
- Updated `utils/gen-snapshot-tests.js` to reflect these changes.
- Removed `babylon` as a devDependency.

### Test plan
1. `yarn install`
2. `yarn run gen-snapshot-tests`
3. `yarn test`
4. Make sure all the tests pass.

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 7c2e3d869a6dd4a97041608c09a0b64bc7372e1f

https://deploy-preview-454--wonder-blocks.netlify.com

File: package.json [hunk 1]
```diff
@@ -51,7 +51,6 @@
     "babel-jest": "^23.6.0",
     "babel-loader": "^8.0.0-beta.0",
     "babel-plugin-dynamic-import-node": "^2.1.0",
-    "babylon": "^6.18.0",
```
Comment (jandrade) [inline package.json]: Removed it to use `@babel/core` instead.

File: packages/wonder-blocks-breadcrumbs/docs.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ```js
-const Link = require("@khanacademy/wonder-blocks-link").default;
-const {Breadcrumbs, BreadcrumbsItem} = require("@khanacademy/wonder-blocks-breadcrumbs");
+import Link from "@khanacademy/wonder-blocks-link";
```
Comment (jeresig) [inline packages/wonder-blocks-breadcrumbs/docs.md]: Yayyy!

File: packages/wonder-blocks-breadcrumbs/generated-snapshot.test.js [hunk 3]
```diff
@@ -8,17 +8,14 @@ import renderer from "react-test-renderer";
 
 // Mock react-dom as jest doesn't like findDOMNode.
 jest.mock("react-dom");
-import Breadcrumbs from "./components/breadcrumbs.js";
-import BreadcrumbsItem from "./components/breadcrumbs-item.js";
+import Link from "@khanacademy/wonder-blocks-link";
```
Comment (jeresig) [inline packages/wonder-blocks-breadcrumbs/generated-snapshot.test.js]: Ahh, very cool!

File: utils/gen-snapshot-tests.js [hunk 4]
```diff
@@ -44,59 +177,101 @@ function generateTestFile(root, examples, componentFileMap) {
...
+
+            const newAst = {
+                type: "File",
+                program: {
+                    type: "Program",
+                    body: statements,
+                },
+            };
+
+            // // Traverse the tree
```
Comment (jeresig) [inline utils/gen-snapshot-tests.js]: Extra comment?

File: utils/gen-snapshot-tests.js [hunk 5]
```diff
@@ -44,59 +177,101 @@ function generateTestFile(root, examples, componentFileMap) {
 
     const modName = root.split("/")[1];
 
+    // tests found on the current file
+    const tests = [];
+
+    // store a reference of the unique declarations
```
Comment (jeresig) [inline utils/gen-snapshot-tests.js]: Might be good to clarify that this is unique imports in the file.

File: utils/gen-snapshot-tests.js [hunk 6]
```diff
@@ -44,59 +177,101 @@ function generateTestFile(root, examples, componentFileMap) {
...
-
+    lines.push(...tests);
     lines.push("});\n");
 
     const data = lines.join("\n");
 
     const outPath = path.join(root, "generated-snapshot.test.js");
     fs.writeFileSync(outPath, data, "utf8");
+
+    const [s, ns] = process.hrtime(hrstart);
```
Comment (jeresig) [inline utils/gen-snapshot-tests.js]: Ooh - that's handy!

File: styleguide.config.js [hunk 7]
```diff
@@ -87,7 +87,10 @@ module.exports = {
         {
             name: "Grid",
             content: "packages/wonder-blocks-grid/docs.md",
-            components: "packages/wonder-blocks-grid/components/*.js",
```
Comment (jeresig) [inline styleguide.config.js]: Why did you get rid of this?

Comment (jandrade) [reply styleguide.config.js]: I changed it b/c locally we were displaying the `Gutter` section twice (one time here, and the other time inside `Internal Components`. But now that you mention it, we are displaying that section for just for documentation purposes.

I'm going to revert this change, and get rid of the `sections` key instead.

File: utils/gen-snapshot-tests.js [hunk 8]
```diff
@@ -44,59 +177,101 @@ function generateTestFile(root, examples, componentFileMap) {
...
+    const ms = ns / 1000000;
+    const kind = ms > 200 ? "yellow" : "green";
```
Comment (jandrade) [inline utils/gen-snapshot-tests.js]: I put this time (200ms) here without any data to back that up, but I'm wondering if there is some data to know the "average" execution speed time for a particular script.

Comment (jeresig) [reply utils/gen-snapshot-tests.js]: Hehe, I guess you could look at the results and make a best guess? If you think this is good then that's fine!

**Pull request merged**